spec/integ/crypto/crypto.spec.ts

```diff
@@ -2283,6 +2284,20 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
     });
 
     describe("Secret Storage and Key Backup", () => {
+        /**
+         * The account data events to be returned by the sync.
+         * Will be updated when fecthMock intercepts calls to PUT `/_matrix/client/v3/user/:userId/account_data/`.
+         * Will be used by `sendSyncResponseWithUpdatedAccountData`
+         */
+        let accountDataEvents: Map<String, any>;
+
+        beforeEach(async () => {
+            createSecretStorageKey.mockClear();
```

Comment:
this is a weird forward reference to `createSecretStorageKey` before it is declared. It would be normal to put the declaration *before* `beforeEach`.

Reply:
sure